pages/pipelines/dependencies.md.erb

```diff
@@ -0,0 +1,65 @@
+# Managing Step Dependencies
+
+- all steps in pipelines have implicit dependencies
+- if you need to change the dependency structure of your steps, you can define explicit dependencies with the `depends_on` attribute
+
+<%= toc %>
+
+## implicit dependencies with wait and block
+
+- wait and block steps provide an implicit dependency structure to your pipeline
+- a wait step, as in the example below, is dependent on all previous steps completing successfully
+- all steps following the wait step are dependent on the wait step
+
+```yml
+steps:
+  - command: "a.sh"
+  - command: "b.sh"
+  - wait
+  - command: "c.sh"
+  - command: "d.sh"
+```
+
+- block steps perform the same function, as well as requiring the unblocking either manually or via an API call
+
+- if you are collecting information with your block steps using the `prompt` or `fields` attributes, but don't require the actual blocking behaviour, you can use an input step
```

Comment:
Arguably the "blocking" part of the behaviour remains -- it will stop its dependents from running until the input has been supplied -- it's just the vacuuming up of implicit dependencies that's removed. (I suspect I'm just picking on an ambiguously-worded notation here, but figure I might as well flag it just in case.)